src/components/ListForm.js

```diff
@@ -27,7 +27,7 @@ const Form = () => {
     const newBook = {
       id: uuidv4(),
       title: formState.title[0],
-      author: formState.title[0],
+      author: formState.author[0],
       categories: formState.category[0],
     };
 
```

Comment:
According to the project's requirements:

![image](https://user-images.githubusercontent.com/57421823/166402595-f397bd49-c759-495a-b74f-84ed36de2553.png)

Therefore:

- [ ] Please, ensure that in your `Categories` component there is a button for checking the status.  
This button should use the function `checkCategoriesStatus` declared in the component.

## Explanation and Hints

The function `checkCategoriesStatus` should dispatch the `underConstruction` action creator.

After doing this, when clicking this button should fire the `checkCategoriesStatus`, triggering the `underConstruction` action creator, showing the text "under construction".

The names of the function and the action creator can be different if you wish.

If you implement this, we will meet this requirement 💪